README.md

```diff
@@ -1 +1,18 @@
-# hw01
\ No newline at end of file
+# hw01
+## Biography of Yunyan Mo
+
+My name is Yunyan Mo. I usually ask people to just call me *Yan* because it is easier to prounce than my full name and will allieviate the inconviniences and embarassments one might feel when I was first introduced to him/her. 
+
+Given my unique name, it is not hard to deduce where did I come from. I was born and raise in **Guangdong, China**. It is a beautiful southern province. If you want to learn more about my hometown, you can visit [this link.](https://en.wikipedia.org/wiki/Guangdong)
+
+I enjoy reading, drawing, and watching movies. Here are some movies that I plan to watch over the summer:
+* Thelma and Louise
+* Paprika
+* 2001: A Space Odyssey
+* Baryy Lyndon
+* Loving Vincent
+* Memoirs of a Geisha
+
+![This is me in Kyoto, Japan, I look forward to the day when we can travel around the world freely again.](https://scontent-ort2-1.xx.fbcdn.net/v/t1.0-9/70931713_107528367318081_2020253329279418368_o.jpg?_nc_cat=102&_nc_sid=09cbfe&_nc_ohc=yMLF4sFhvukAX-i5OLy&_nc_ht=scontent-ort2-1.xx&oh=61ae425a6ba83ccf92219b7f5eaacbdf&oe=5F1983C0)
```

Comment:
This isn't quite the right way to embed images. It doesn't appear in your repo and doesn't appear for me either. Not sure why. What you can do is you can upload the image to your RStudio server and commit that change to GitHub and then you can reference it normally by the filename!